content/pages/changelog/index.md

```diff
@@ -13,6 +13,267 @@ For historical changelog entries, please see our
 [2019](page://changelog/2019) and
 [2018 release notes](page://changelog/2018).
 
+## 2020-06-11 / New enterprise events for tasks
+
+Starting today, the [enterprise event stream](g://events/for-enterprise/) will
+begin producing new task and task assignment events, and some existing task
+events will return additional fields.
+
+### New task and task assignment events
+
+#### Task update
+
+```js
+      {
```

Comment:
Do you think it might make sense to extract this into its own section like we did with Shield events? We can then document just the `additional_details` parts like we did with those, as the rest of each event is mostly identical between events?

Reply:
I am not clear on what "extract each of these into its own section" would entail

Reply:
Basically, can we follow what we did with the Shield events? We list for each event what the additional details are, and we omit the part of the event that's generic to all events.

https://developer.box.com/guides/events/shield-alert-events/

Reply:
I still think it would be nicer to extract the example event payloads into their own section of the guides, and then make this changelog entry very short and concise.

Reply:
Example of what I would like this changelog to read as instead.

===

Starting today, the [enterprise event stream](g://events/for-enterprise/) will
begin producing new task and task assignment events, and some existing task
events will return additional fields.

## New events

* A new `TASK_UPDATE` event is triggered when a task is updated
* A new `TASK_ASSIGNMENT_DELETE` event is triggered when a task is unassigned from a user

## Updated events

### Change to `TASK_CREATE` 

When a task is created, the event now includes the task's ID (`task.id`), the ID of the user who created the task (`task.created_by.id`), the task's description (`task.message`), and the optional due date of the task (`task.due_date`) within the `additional_details` object.

```json
"task": {
  "id": "1234567",
  "due_at": "2020-07-06T10:49:44-07:00",
  "message": "task description",
  "created_by": {
    "id": 123456,
    "login": "email@example.com"
  }
}
```

## Change to `TASK_ASSIGNMENT_CREATE` and `TASK_ASSIGNMENT_UPDATE`

When a task assignment is created or updated, the event now includes the task's ID (`task.id`), the ID of the assigned user (`task_assignment.assigned_to.id`) and their login (`task_assignment.assigned_to.login`), the task's description (`task.message`), and the optional due date of the task (`task.due_date`) within the `additional_details` object.

```json
"additional_details": {
  "task": {
    "id": "1234567",
    "created_by": {
      "id": 12345,
      "login": "email@example.com"
    }
  },
  "task_assignment": {
    "assigned_to": {
      "id": 12346,
      "login": "email+2@example.com"
    },
    "status": "NOT_STARTED",
    "message": "assignee message"
  },
  ...
}
```

See the task events documentation
for more information on the payloads produced within these events.